changelog.md

```diff
@@ -4,10 +4,11 @@
 
 ### 0.14.2
 
-* Moved `@types/chalk` from dependencies to devDependencies - orta
+* Replaced deprecated `lodash.isarray` package with `Array.isArray` - damassi
```

Comment:
Looking for confirmation from @orta on shipping 0.14.2 or holding off on it for anything else to keep this changelog message here. Otherwise we usually add changelog messages right below the `//  Add your own contribution below` line and then add the version number we are shipping to the changelog as we package up a release.